dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/hadoop/BashTask.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2022-2024 Google LLC
+ * Copyright 2013-2021 CompilerWorks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.edwmigration.dumper.application.dumper.connector.hadoop;
+
+import static java.lang.String.format;
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+import com.google.common.io.ByteSink;
+import com.google.common.io.Files;
+import com.google.common.io.Resources;
+import com.google.edwmigration.dumper.application.dumper.io.OutputHandle;
+import com.google.edwmigration.dumper.application.dumper.task.Task;
+import com.google.edwmigration.dumper.application.dumper.task.TaskCategory;
+import com.google.edwmigration.dumper.application.dumper.task.TaskRunContext;
+import java.io.File;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.io.Writer;
+import java.net.URL;
+import java.time.Duration;
+import org.apache.commons.exec.CommandLine;
+import org.apache.commons.exec.DefaultExecutor;
+import org.apache.commons.exec.ExecuteException;
+import org.apache.commons.exec.ExecuteWatchdog;
+import org.apache.commons.exec.PumpStreamHandler;
+
+public class BashTask implements Task<Void> {
+
+  private static final Duration SCRIPT_TIMEOUT = Duration.ofMinutes(5);
+
+  private final String scriptName;
+
+  public BashTask(String scriptName) {
+    this.scriptName = scriptName;
+  }
+
+  @Override
+  public String getTargetPath() {
+    return scriptName + ".out";
+  }
+
+  private void doRun(ByteSink outputSink, ByteSink errorSink, ByteSink exitStatusSink)
+      throws IOException {
+    String scriptFilename = scriptName + ".sh";
+    URL resourceUrl = Resources.getResource("hadoop-scripts/" + scriptFilename);
+    byte[] scriptBody = Resources.toByteArray(resourceUrl);
+    File scriptDir = new File("dwh-migration-tools-tmp");
```

Comment:
What is the reason for copying scripts into the new location?

Reply:
It is an extraction from jar to the local file system.

Reply:
I've prepared the changes in PR #425.